domains/authorization/Proxy.IntegrationTests/ProxyTests.cs

```diff
@@ -161,10 +162,11 @@ public async Task GivenB2C_WhenV20240515GetEndpointsAreUsed_ThenAppendQueryParam
     {
         // Arrange
         var orgIds = new List<string> { Guid.NewGuid().ToString() };
+        var mockEndpoint = $"/wallet-api/v1{endpoint.Remove(0, 11)}";
```

Comment:
why the remove part? Can't you remove it from the InlineData? Or add /v1 to them?

Reply:
We still use the endpoints to call the method in the controllers